cllc-public-app/cllc-public-app.csproj

```diff
@@ -45,11 +45,19 @@
 
   <ItemGroup>
     <!-- Don't publish the SPA source files, but do show them in the project files list -->
+    <Compile Remove="Test\**" />
     <Content Remove="$(SpaRoot)**" />
+    <Content Remove="Test\**" />
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
   </ItemGroup>
   
 
+  <ItemGroup>
```

Comment:
FYI folks - we should be able to switch to Dotnet Core 2.1 in OpenShift soon.  It's been out for a while now.  I gather the reason why you exclude this directory is because it is using 2.1?

Reply:
I discovered that this was done automatically by VS when I previously got the latest code, not sure why